nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
+
+    def add(self, *args, **kwargs):
+        self.args += args
+        self.kwargs.update(OrderedDict(kwargs))
```

Comment:
One thing that's a little disconcerting is that in theory this could be called after `process`, and then a user might be confused why it doesn't just "work". My suggestion is to check if `self.actions is None`, otherwise raise a warning that says the added action will not be in the model until the next 'whenever'.